### PR TITLE
scripts: new backport-create-issue script

### PR DESCRIPTION
**Pull request by smithfarm**
This is an initial (and rather careless) standalone implementation of a script written by Loic Dachary. It has been tested manually, and seems to do the job.

The script iterates over all issues in status "Pending Backport" and, where necessary, creates issues in the Backport tracker and "Copied to/from" relations between the backport tracker issue and the parent issue.

I only tagged @ktdreyer and @theanalyst for review, but all reviews are welcome!!!

### CODE REVIEW

File: src/script/backport-create-issue [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+            continue
+        if len(issue['backports']) == 0:
+            logging.error(url(issue) + " the backport field is empty")
+        update_relations(r, issue, dry_run)
+    logging.info("Processed {} issues with status Pending Backport"
+                 .format(counter))
+    return None
+
+
+args = parse_arguments()
```
Comment (tchaikov) [inline src/script/backport-create-issue]: might want to guard following statements with 

```py
if __name__ == '__main__':
```

it's a useful idiom in python development. it helps when one wants to debug part of this script without running all the main() statements in a wholesale.

File: src/script/backport-create-issue [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+            continue
+        if not set_backport(issue):
+            logging.error(url(issue) + " no backport field")
+            continue
+        if len(issue['backports']) == 0:
+            logging.error(url(issue) + " the backport field is empty")
+        update_relations(r, issue, dry_run)
+    logging.info("Processed {} issues with status Pending Backport"
+                 .format(counter))
+    return None
```
Comment (tchaikov) [inline src/script/backport-create-issue]: this return value is never used. and a method return nothing actually evaluates to `None`. so this is not necessary.

File: src/script/backport-create-issue [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+    parser.add_argument("--dry-run", help="Do not write anything to Redmine",
+                        action="store_true")
+    return parser.parse_args()
+
+def set_logging_level(a):
+    if a.debug:
+        logging.basicConfig(level=logging.DEBUG)
+    else:
+        logging.basicConfig(level=logging.INFO)
+    return None
```
Comment (tchaikov) [inline src/script/backport-create-issue]: no need to return "None".

File: src/script/backport-create-issue (hunk 2 above)
Comment (smithfarm) [reply src/script/backport-create-issue]: I prefer to make the return value explicit. When I'm re-reading the code, it signals to me that this function is not designed to return anything useful.

Comment (theanalyst) [reply src/script/backport-create-issue]: While I support not returning, I'm ok just merging this as is as this should beat copying issues in tracker that many users get wrong anyway

Comment (smithfarm) [reply src/script/backport-create-issue]: @theanalyst Thanks! The reasoning for making the return value explicit is (a) it does something useful (remind me that the function is only used for its side effects, not for its return value), and (b) it does no harm. But this is a matter of coding style preference.

**Pull request merged**